features/dish_selecting.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Hungry User selects the Dish
+  In order to select a Dish
+  As Hungry User
+  I should be able to view available Dishes in Menu of Caterer
+
+  Scenario: Hungry User selects from Internal Menu
+    Given selected Caterer has Internal Menu
+    And there is at least one Dish in it
+    When I try to select a Dish from Internal Menu
+    Then I should have that Dish selected as mine
+
+  Scenario: Hungry User selects from External Menu
+    Given selected Caterer has External Menu
+    When I try to input selected Dish manually
+    And I provided name and price of that Dish
+    Then I should have that Dish selected as mine
```

Comment:
Given There is open order for "Korova" caterer
And "Korova" caterer has external menu
When I add name and price of dish
Then  I should have that dish selected as mine in order